test/e2e/prometheus_test.go

```diff
@@ -370,6 +371,95 @@ func TestBodySizeLimit(t *testing.T) {
 	}
 }
 
+func TestPrometheusPolicyHeaders(t *testing.T) {
+	const testAccount = "test-policy-headers"
+	const testNs = "test-policy-headers-ns"
+
+	ns := &v1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: testNs,
+			Labels: map[string]string{
+				framework.E2eTestLabelName: framework.E2eTestLabelValue,
+			},
+		},
+	}
+	ns, err := f.KubeClient.CoreV1().Namespaces().Create(ctx, ns, metav1.CreateOptions{})
+	if err != nil {
+		t.Fatal(err)
+	}
+	t.Cleanup(func() {
+		foreground := metav1.DeletePropagationForeground
+		if err := f.KubeClient.CoreV1().Namespaces().Delete(ctx, testNs, metav1.DeleteOptions{PropagationPolicy: &foreground}); err != nil {
+			t.Logf("err deleting namespace %s: %v", testNs, err)
+		}
+	})
+
+	err = framework.Poll(2*time.Second, 10*time.Second, func() error {
+		_, err := f.CreateServiceAccount(testNs, testAccount)
+		return err
+	})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	err = framework.Poll(2*time.Second, 10*time.Second, func() error {
+		_, err = f.CreateClusterRoleBinding(testNs, testAccount, "admin")
```

Comment:
this role binding should be cleaned up. or reuse the e2e test account "cluster-monitoring-operator-e2e" 
created in https://github.com/openshift/cluster-monitoring-operator/blob/master/test/e2e/framework/framework.go#L184

Reply:
Done, Fixed.